R/environment.R

```diff
@@ -85,19 +89,22 @@
 #' @md
 r_environment <- function(name, version = NULL,
                           environment_variables = NULL,
+                          rscript_path = NULL,
```

Comment:
can we also test/verify this with a local run?

Reply:
yup, all good.